infrastructure/cdn-in-a-box/cache/uname

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -o errexit -o nounset
+get_rhel_version() {
+	local redhat_release=/etc/redhat-release
+	local default_version="${RHEL_VERSION:-8}"
+	if [ -e $redhat_release ]; then
+		releasever="$(rpm -q --qf '%{version}' -f $redhat_release)"
+		releasever="${releasever%%.*}"
+	else
+		releasever=${default_version}
+	fi;
+
+	echo "el${releasever}"
+}
+rhel_version="$(get_rhel_version)"
+kernel_version="3.10.0-862.11.6.${rhel_version}"
```

Comment:
Do we need to spoof the kernel version? Things that need to interact with the kernel will actually be interacting with the host kernel, won't they?

Reply:
Not for `t3c`, we don't. In `traffic_ops_ort.pl`, we did:

https://github.com/apache/trafficcontrol/blob/a289d9ce008df420b41f4b3ac488e094860b10f0/traffic_ops_ort/traffic_ops_ort.pl#L343-L348

Removed the `uname` shim in 549aa07ce5.